### PR TITLE
[#4270] Add tests for DevToolsUtils

### PR DESCRIPTION
**Pull request by stratigos**
Associated Issue: #4270

### Summary of Changes

* Add tests for functions in `src/utils/DevToolsUtils.js`

### CODE REVIEW

**Comment by stratigos** [comment]
I suspect the following error reported by travis isnt related to this PR:

`error An unexpected error occurred: "https://registry.yarnpkg.com/xdg-basedir/-/xdg-basedir-3.0.0.tgz: Request failed \"502 Bad Gateway\"".`

Could someone re run the travis-ci check for me?

File: src/utils/tests/DevToolsUtils.spec.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import { reportException, executeSoon } from "../DevToolsUtils.js";
+
+describe("DevToolsUtils", () => {
+  describe("reportException", () => {
+    beforeEach(() => {
+      global.console = { error: jest.fn() };
+    });
```
Comment (jasonLaster) [inline src/utils/tests/DevToolsUtils.spec.js]: do we need to unmock these in an `afterEach`?

Comment (jasonLaster) [reply src/utils/tests/DevToolsUtils.spec.js]: closing because this works.

Comment (stratigos) [reply src/utils/tests/DevToolsUtils.spec.js]: Thanks for the tip, Ill keep that in mind for the future

**Pull request merged**